packages/skill-lesson/video/video.tsx

```diff
@@ -36,6 +37,7 @@ export const Video: React.FC<
       exerciseOverlayRenderer = () => <DefaultOverlay />,
       defaultOverlayRenderer = () => <DefaultOverlay />,
       blockedOverlayRenderer = () => <BlockedOverlay product={product} />,
+      finishedSectionOverlayRenderer = () => <FinishedSectionOverlay />,
```

Comment:
needed to be able to override `FinishedSectionOverlay` so added as optional prop to `Video`